test/core/rules/L001_test.py

```diff
@@ -0,0 +1,25 @@
+# flake8: noqa
+"""Tests for L001."""
+from .std_test import rules__test_helper
+
+
+test_cases = {
+    "pass": [],
+    "fail": [
+        {
+            "query":
+"""
+SELECT 1     
+""",
```

Comment:
This pattern with `"""` strings makes sense if it's required, but I think using normal string and then escaping the newlines, i.e. "\nSELECT 1\n" is actually more explicit and easier to read.

Reply:
It is for short tests, but as soon as we get into tests with CTE's and several lines of statements it gets very confusing. I'll have a think about how to get a solution where adding a new test case requires minimal boiler plate and is easy to read